### PR TITLE
Fixes bug for inflammatory typo

### PR DESCRIPTION
**Pull request by mayya-bond-kc**
## Describe your changes

bugfix for api tests

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/src/server/app/validation/validate-request.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const incompleteWithoutReasons = function (body) {
 };
 
 const allArrayElementsInArray = function(arrayToCheck, arrayToCheckAgainst) {
+	console.log(_.difference(arrayToCheck, arrayToCheckAgainst));
```
Comment (james-hill-kc) [inline apps/api/src/server/app/validation/validate-request.js:28]: Was this left in intentionally?

**Review by james-hill-kc — DISMISSED** [review]

File: apps/api/src/server/app/validation/validate-request.js (hunk 1 above)
Comment (mayya-bond-kc) [reply apps/api/src/server/app/validation/validate-request.js:28]: no :(

**Review by james-hill-kc — APPROVED** [review]

**Pull request merged**